File: CS-Examples/06_Text/CreateBarcode.cs
using System;
using System.Drawing;
using System.Windows.Forms;
using Spire.Doc;
using Spire.Doc.Documents;
using Spire.Doc.Fields;

namespace CreateBarcode
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
           // Create a new instance of the Document class.
			Document doc = new Document();

			// Add a new section to the document and get its first paragraph.
			Paragraph p = doc.AddSection().AddParagraph();

			// Append the text "H63TWX11072" to the paragraph and obtain the TextRange object.
			TextRange txtRang = p.AppendText("H63TWX11072");

			// Set the font name for the text range to "C39HrP60DlTt".
			txtRang.CharacterFormat.FontName = "C39HrP60DlTt";

			// Set the font size for the text range to 80.
			txtRang.CharacterFormat.FontSize = 80;

			// Set the text color for the text range to SeaGreen.
			txtRang.CharacterFormat.TextColor = Color.SeaGreen;

			// Specify the file name for the resulting document.
			string output = "CreateBarcode.docx";

			// Save the document to a file with the specified file name and format (Docx).
			doc.SaveToFile(output, FileFormat.Docx);

			// Clean up resources used by the document.
			doc.Dispose();
			
            Viewer(output);
        }
        private void Viewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB-Examples/06_Text/CreateBarcode.vb
Imports Spire.Doc
Imports Spire.Doc.Documents
Imports Spire.Doc.Fields

Namespace CreateBarcode
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Create a new Document object
			Dim doc As New Document()

			' Add a section to the document and get a reference to the newly added paragraph
			Dim p As Paragraph = doc.AddSection().AddParagraph()

			' Create a TextRange object and set its text to "H63TWX11072"
			Dim txtRang As TextRange = p.AppendText("H63TWX11072")

			' Set the font name of the text to "C39HrP60DlTt"
			txtRang.CharacterFormat.FontName = "C39HrP60DlTt"

			' Set the font size of the text to 80
			txtRang.CharacterFormat.FontSize = 80

			' Set the text color of the text to SeaGreen
			txtRang.CharacterFormat.TextColor = Color.SeaGreen

			' Specify the output file name as "CreateBarcode.docx"
			Dim output As String = "CreateBarcode.docx"

			' Save the document to the specified file in Docx format
			doc.SaveToFile(output, FileFormat.Docx)

			' Dispose the Document object to release resources
			doc.Dispose()

			Viewer(output)
		End Sub
		Private Sub Viewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace
